doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,117 @@
+
+.Credit Card Dispute project
+To help with this decisioning, Fortress Bank used {CENTRAL} to create the *CreditCardDisputeCase* project that contains the *fraudDispute* business process that models the dispute process.
+
+image:ai/casefile.png[]
+
+.Process variables
+When Joe reported the dispute, an instance of the *fraudDispute* process was created with case ID `FR-00000004`.  The *Process Variables* tab contains several variables and values specific to Joe's account, including `CaseID`, `caseFile_cardholderRIskRating` (the creditcard holder risk rating), and `caseFile_disputeRiskRating` (the risk rating of this dispute):
+
+image:ai/process-variables.png[]
+
+The table also has the `casefile_automated` variable with the value `true`. This indicates that the dispute met the criteria to be automatically processed.
+
+.Process diagram
+The *Diagram* tab contains the BPMN diagram that shows the decision paths that the bank uses when deciding whether to process a dispute automatically or manually:
+//<IMAGE 2:31>
+
+image:ai/bpmn.png[]
+
+The *Decision Task* task contains the rules that determine whether the dispute will be automatically charged back to Joe's account (*Milestone 1*) or whether more investigation is required (*Milestone 2*) based on the values of the `caseFile_cardholderRiskRating` and the `caseFile_disputeRiskRating` variables. If Joe's dispute matches the criteria for automatic approval, Milestone 1 is followed and the disputed amount is charged back to his account. If Joe's dispute requires manual evaluation, the process in Milestone 2 is started.
+//COMMENT: Need image that shows all of milestone 2.
+//image:ai/automatic.png[]
+//image:ai/milestone2.png[]
+//3:15
+
+//In this case, the Decision Task task decided to process Joe's dispute automatically so it followed *Milestone 1: Automatic Chargeback*.
+
+.DMN model
+The following simplified DMN model shows the decision process that is part of the *fraudDispute* *Decision Task* task:
+//<IMAGE> 5:13>
+
+image:ai/dmn.png[]
+
+The input variables are *Age* (cardholder age), *Incident Count* (the number of previous disputes for this cardholder), *Cardholder Status*, and *Fraud Amount*.
+
+{PRODUCT} uses input variables in decision tables to support digital decisioning. The decision tables are created by human business analysts. The data is provided in spreadsheet files that are uploaded to the {CENTRAL} project. The Fortress Bank Credit Card Dispute process has two decision tables, the *Cardholder Risk Rating* table and the *Dispute Risk Rating* table. The *Cardholder Risk Rating* table contains three input variables:  *Incident Count*, *Cardholder Status*, and *Age*. The *Dispute Risk Rating* table contains the *Cardholder Status* input variable. It calculates the risk of the dispute based on the cardholder status and the amount of the dispute.
```

Comment:
Not sure what the phrase `The data is provided in spreadsheet files that are uploaded to the {CENTRAL} project ` is technically referring to, here.

```suggestion
{PRODUCT} uses input variables in decision tables to support digital decisioning. The decision tables are created by human business analysts. The Fortress Bank Credit Card Dispute process has two decision tables, the *Cardholder Risk Rating* table and the *Dispute Risk Rating* table. The *Cardholder Risk Rating* table contains three input variables:  *Incident Count*, *Cardholder Status*, and *Age*. The *Dispute Risk Rating* table contains the *Cardholder Status* input variable. It calculates the risk of the dispute based on the cardholder status and the amount of the dispute.
```

Reply:
To add: maybe the analyst first circulates the Business Requirement Analysis document in the form of Words and Excel with all the stakeholder, to make sure the policy is accepted.
But at the end of the day, when authoring the DMN model, it will be designed "manually" by using the DMN editor, not via any file upload to BC.